MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -8570,4 +8601,87 @@ public boolean checkYearlyRetirements() {
         }
         return false;
     }
+
+    @Override
+    public int getTechIntroYear() {
+        if (campaignOptions.limitByYear()) {
+            return calendar.get(Calendar.YEAR);
+        } else {
+            return 0;
+        }
+    }
```

Comment:
I don't see this method being used anywhere, so this may not matter, but should we use a named constant rather than just 0 if limitByYear is turned off?

Reply:
Now that I look at this I see I got the logic backwards. The zero is there because it's a literal zero. Any tech that is introduced at a year >= 0 is legal. But it's actually the other way around, and it's only legal if 0 >= the intro year, which is not going to work.

Reply:
Yup, that's what I did. Changed to Integer.MAX_VALUE, which will work until the in-game universe gets a bit past the year two billion.